website/docs/guides/flagger-manual-gating.mdx

```diff
@@ -0,0 +1,128 @@
+your own gates.
+
+## Pre-requisites
+- Basic knowledge of [Flagger](https://flagger.app).
+- An existing `Canary` object and target deployment.
+- Flagger's load tester [installed](https://docs.flagger.app/usage/webhooks#load-testing)
+
+## Basic introduction to Webhooks and Gating
+Flagger can be configured to work with several types of hooks which will be called at 
+given stages during a progressive delivery rollout. Some of these allow you to manually 
+gate whether a rollout proceeds at certain points:
+- Before a new deployment is scaled up and canary analysis begins with `confirm-rollout`. 
+- Before traffic weight is increased with `confirm-traffic-increase`.
+- Before a new version is promoted following successful canary analysis with `confirm-promotion`.
+
+Any URL can be used as a webhook target, if will approve if it returns with a
+`200 OK` status code, and halt if it's `403 Forbidden`.
+
+The webhook will receive a JSON payload that can be unmarshaled as
+`CanaryWebhookPayload`:
+
+```go
+type CanaryWebhookPayload struct {
+	// Name of the canary
+	Name string `json:"name"`
+
+	// Namespace of the canary
+	Namespace string `json:"namespace"`
+
+	// Phase of the canary analysis
+	Phase CanaryPhase `json:"phase"`
+
+	// Metadata (key-value pairs) for this webhook
+	Metadata map[string]string `json:"metadata,omitempty"`
+}
+```
+
+For more information on Webhooks in Flagger, see the 
+[Flagger documentation](https://docs.flagger.app/usage/webhooks)
+
+
+## Using Flagger's load tester to manually gate a promotion
+To enable manual approval of a promotion we are going to configure the 
+`confirm-promotion` webhook to call a particular gate provided through 
+Flagger's included load tester. This is an easy way to experiment with 
+the capability using Flagger's included components. 
+
+**Important note**
+We strongly recommend that you DO NOT USE the load tester for manual gating 
+in a production environment. There is no auth on the load tester, so 
+anyone with access to the cluster would be able to open and close; and 
+the load tester has no storage, so if restarted - all gates would close. 
+
+Instead, configure these webhooks for appropriate integration with a 
+tool of your choice such Jira, Slack, Jenkins, etc.
+
+### Configure the confirm-promotion webhook
+In your Canary object, add the following in the `analysis` section:
+
+```yaml
+  analysis:
+    webhooks:
+      - name: "ask for confirmation"
+        type: confirm-promotion
+        url: http://flagger-loadtester.test/gate/check
+```
+
+This gate is closed by default.
+
+### Deploy a new version of your application
+Trigger a Canary rollout by updating your target deployment/daemonset, for 
+example by bumping the container image tag. A full list of ways to trigger 
+a rollout is available 
+[here](https://docs.flagger.app/faq#how-to-retry-a-failed-release).
+
+### Wait for the Canary analysis to complete
+TODO: add screenshots and explain with WGE UI
```

Comment:
@sympatheticmoose let's add screenshots or remove this TODO

Reply:
I want the screenshots. Means we can show the UI. Just havent got to it